process/graduation_criteria.adoc

```diff
@@ -18,6 +18,8 @@ To be accepted to incubating stage, a project must meet the sandbox stage requir
  * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria
+ * A clear versioning scheme.
```

Comment:
"clear" is kind of subjective. What is the versioning scheme of `grep`? What is the ask here? Guideline? Links to examples? Is this looking for semver-style compatibility guarantees?

Reply:
I will ask the TOC for examples, but I'm assuming a documented release process and versioning scheme is all that's required